### ISSUE TITLE
Tranquil Clouds: Neural Networks for Learning Temporally Coherent Features in Point Clouds

### ISSUE THREAD

**Issue by j20232** [j20232/survey#52]
![image](https://user-images.githubusercontent.com/13940471/74333244-e6e1fa00-4dda-11ea-8bcd-1b39cb28e830.png)

![image](https://user-images.githubusercontent.com/13940471/74333286-fbbe8d80-4dda-11ea-9073-aedf4e318e7a.png)

## Summary
- Permutation invariant loss terms for temporally coherent point set generation
- A Siamese training setup and generator architecture for point-based super-resolution with neural networks
- Enabling improved output variance by allowing for dynamic adjustments of the output size

## Abstract

Point clouds, as a form of Lagrangian representation, allow for powerful and flexible applications in a large number of computational disciplines. We propose a novel deep-learning method to learn stable and temporally coherent feature spaces for points clouds that change over time. We identify a set of inherent problems with these approaches: without knowledge of the time dimension, the inferred solutions can exhibit strong flickering, and easy solutions to suppress this flickering can result in undesirable local minima that manifest themselves as halo structures. We propose a novel temporal loss function that takes into account higher time derivatives of the point positions, and encourages mingling, i.e., to prevent the aforementioned halos. We combine these techniques in a super-resolution method with a truncation approach to flexibly adapt the size of the generated positions. We show that our method works for large, deforming point sets from different sources to demonstrate the flexibility of our approach.

## Author

- <a href="https://arxiv.org/search/cs?searchtype=author&amp;query=Prantl%2C+L">Lukas Prantl</a>
- <a href="https://arxiv.org/search/cs?searchtype=author&amp;query=Chentanez%2C+N">Nuttapong Chentanez</a>
- <a href="https://arxiv.org/search/cs?searchtype=author&amp;query=Jeschke%2C+S">Stefan Jeschke</a>
- <a href="https://arxiv.org/search/cs?searchtype=author&amp;query=Thuerey%2C+N">Nils Thuerey</a>

## Subjects

- `cs.CV`: Computer Vision and Pattern Recognition
- `cs.GR`: Graphics
- `cs.LG`: Machine Learning
- `stat.ML`: Machine Learning

## Comment

Further information and videos at <a class="link-external link-https" href="https://ge.in.tum.de/publications/2020-iclr-prantl/" rel="external noopener nofollow">this https URL</a>

## Link

- arXiv: https://arxiv.org/abs/1907.05279
- Project: https://ge.in.tum.de/publications/2020-iclr-prantl/
- Code: https://gitlab.com/Prantl/NeuralParticles
